libraries/AP_AHRS/AP_AHRS.h

```diff
@@ -596,6 +591,11 @@ class AP_AHRS
     AP_Int8 _gps_minsats;
     AP_Int8 _gps_delay;
     AP_Int8 _ekf_type;
+    AP_Float _rot_roll;
```

Comment:
better to rename these to _custom_roll etc

Reply:
done